docs/modules/ROOT/pages/release-notes/README.adoc

```diff
@@ -8,6 +8,7 @@ Detailed information about features and fixes in every release.
 
 *2021*
 
+* 2021 May - xref:release-notes/release-notes-28-1.adoc[Payara Enterprise 5.28.1]
```

Comment:
Months are wrong. (missed previous update also which is wrong)

5.28.0 -> May 2021
5.28.1 -> June 2021

Reply:
Fixed